### PR TITLE
link to documentation for cross-sub registration

### PR DESCRIPTION
**Pull request by mgreenegit**
## Checklist

- [ ] I have read the [_Submitting Changes_](https://github.com/Azure/azure-powershell/blob/master/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](https://github.com/Azure/azure-powershell/blob/master/CONTRIBUTING.md)
- [ ] The title of the PR is clear and informative
- [ ] The appropriate [change log has been updated](https://github.com/Azure/azure-powershell/blob/master/CONTRIBUTING.md#updating-the-change-log)
- [ ] The PR does not introduce [breaking changes](https://github.com/Azure/azure-powershell/blob/master/documentation/breaking-changes/breaking-changes-definition.md)
- [ ] If applicable, the changes made in the PR have proper test coverage
- [ ] For public API changes to cmdlets:
    - [ ] the changes have gone through a [cmdlet design review](https://github.com/Azure/azure-powershell-cmdlet-review-pr) (Microsoft internal partners only)
    - [ ] the cmdlet markdown files were [generated using the `platyPS` module](https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/help-generation.md)

### CODE REVIEW

File: src/Automation/Automation/help/Register-AzAutomationDscNode.md [hunk 1]
```diff
@@ -24,6 +24,12 @@ Register-AzAutomationDscNode -AzureVMName <String> [-NodeConfigurationName <Stri
 ## DESCRIPTION
 The **Register-AzAutomationDscNode** cmdlet registers an Azure virtual machine as an APS Desired State Configuration (DSC) node in an Azure Automation account.
 
+If you need to register a node to an automation account in a different subscription,
```
Comment (markcowl) [inline src/Automation/Automation/help/Register-AzAutomationDscNode.md]: Why is this the case? ARM supports this fnctionality, and cmdlets could support it if they chose to.

Comment (mgreenegit) [reply src/Automation/Automation/help/Register-AzAutomationDscNode.md]: We will not have capacity for this work during Vb.

**Comment by mgreenegit** [comment]
#sign-off

File: src/Automation/Automation/help/Register-AzAutomationDscNode.md [hunk 2]
```diff
@@ -24,6 +24,12 @@ Register-AzAutomationDscNode -AzureVMName <String> [-NodeConfigurationName <Stri
...
+you will need to use an ARM template rather than cmdlets. 
+See the Azure Automation
+[documentation](https://docs.microsoft.com/en-us/azure/automation/automation-dsc-onboarding#registering-virtual-machines-across-azure-subscriptions)
+for more details.
```
Comment (markcowl) [inline src/Automation/Automation/help/Register-AzAutomationDscNode.md]: Please add a changelog entry for this change in the Automation Changelog

Comment (mgreenegit) [reply src/Automation/Automation/help/Register-AzAutomationDscNode.md]: done

**Pull request merged**